packages/driver/src/cy/ensures.ts

```diff
@@ -20,9 +20,7 @@ export const create = (state: StateFunc, expect: $Cy['expect']) => {
   // into an array and loop through each and verify
   // each element in the array is valid. as it stands
   // we only validate the first
-  const validateType = (subject, type, cmd = state('current')) => {
-    const name = cmd.get('name')
-
+  const validateType = (subject, type, name) => {
```

Comment:
I changed all of these to accept `name`, so that the signatures of the various "ensures" functions match with each other. Name is the only property we read off of `command` anyway. For a while I thought I needed the full command, but that ended up refactored away at some point, and just never got around to reverting the signatures.

The `name` / `command` arguments are new in Cy12, so we can be completely certain no one outside the monorepo is using them.